frontend/app/src/connection/WebsocketConnection.tsx

```diff
@@ -684,15 +684,16 @@ export function doInitPings(
   connect = () => {
     const uriParts = uriPartsList[uriNumber]
     const healthzUri = buildHttpUri(uriParts, SERVER_PING_PATH)
-    const allowedOriginsUri = buildHttpUri(uriParts, ALLOWED_ORIGINS_PATH)
+    const hostConfigUri = buildHttpUri(uriParts, HOST_CONFIG_PATH)
 
     logMessage(LOG, `Attempting to connect to ${healthzUri}.`)
+    logMessage(LOG, `Attempting to connect to ${hostConfigUri}`)
```

Comment:
I don't think there's a point in logging this given we're already logging that we're pinging the health check endpoint above